packages/flutter/lib/src/rendering/semantics.dart

```diff
@@ -312,6 +331,16 @@ class SemanticsNode extends AbstractNode {
     }
   }
 
+  Set<SemanticsTag> _tags = new Set<SemanticsTag>();
+
+  /// Tag the [SemanticsNode] with [tag].
```

Comment:
I don't understand what you mean by your second comment. Tags behave just like flags or actions that you add to a SemanticsNode. What special considerations do you have in mind that we need to mention for tags?